prometheus-exporter/templates/deployment.yaml

```diff
@@ -1,51 +1,73 @@
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
```

Comment:
Why are you creating both a `Deployment` and a `DeploymentConfig` in the same chart?